### PR TITLE
Modified Network view graph

### PR DESCRIPTION
**Pull request by LedaLima**
Altered the parameters for the Force graph to spread the nodes a little bit more on its panel

### CODE REVIEW

File: ui/dns/js/components/NetworkPanel.react.js [hunk 1]
```diff
@@ -145,8 +145,8 @@ function filterDataAndBuildGraph()
 
   // Graph force
   var force = d3.layout.force()
-                          .charge(-Math.round(h * 0.45)) // 0.45 is a magic number for graph styling purposes charge is 45% of the grap height
-                          .linkDistance(Math.round(h * 0.081)) // 0.081 is a magic number for graph styling purposes linkDistance is 8.1% of the graph height
+                          .charge(-Math.round(w * 0.45)) // 0.45 is a magic number for graph styling purposes charge is 45% of the grap height
+                          .linkDistance(Math.round(w * 0.081)) // 0.081 is a magic number for graph styling purposes linkDistance is 8.1% of the graph height
```
Comment (rabarona) [inline ui/dns/js/components/NetworkPanel.react.js]: is w for width? In that case I'd update the comment next to the code line as well.

**Comment by EverLoSa** [comment]
LGTM

**Comment by daortizh** [comment]
LGTM :+1:

**Comment by rabarona** [comment]
L G T M

**Pull request merged**